docs/relying-parties/tutorials/integrating-with-fxa.md

```diff
@@ -74,31 +78,37 @@ Before starting integration, please send a request to fxa-staff[at]mozilla.com t
 
 We communicate with our relying parties via the [firefox-accounts-notices group][firefox-accounts-notices].  Please join this list to avoid any surprises.
 
-## OAuth Integration
+## OpenID Connect Integration
+
+### Relying Party vs. Resource Server
+
+A Relying Party (RP) is an application or website that outsources its user authentication functionality to an OpenID Provider, defined as part of the OpenID specifications. The user will be prompted to login to the Relying Party with their Firefox Account, and is able to see the login as a distinct [Connected Service][connected-services] in FxA Settings. OpenID Connect contains many specifications for a variety of identity exchange and authorization, based on the OAuth 2.0 framework of specifications.
+
+In contrast, a Resource Server is an OAuth 2.0 term for an API server. A Relying Party may also act as a Resource Server, with a user never seeing a direct Relying Party login. Users that access a resource server with OAuth tokens will not see their access as a distinct Relying Party login on the FxA Settings page. Services that are accessed via Firefox (Sync, Relay, etc.) using an existing OAuth token are all examples of Resource Servers. Their access shows up in FxA Settings as the browser session the user is logged into.
```

Comment:
Relay is actually listed as Relay on the connected services.  Sync is only special case.

> using an existing OAuth token

I think the word "existing" here can be confusing.  It reads as if it's referring to some particular existing token.

Reply:
Relay is not always listed anymore as they now operate as a Resource Server. It is now possible to use Relay via Firefox without it appearing in Connected Services.

Reply:
I think reorganizing the first 3 sentences could help clarify.  How does this sound?
> A Relying Party may also act as a Resource Server, an OAuth 2.0 term for an API server.  In that case a user will never see a direct Relying Party login and will not see a distinct row in their Connected Devices list on the FxA Settings page. [Services that are....]

Reply:
@chenba Agreed, rephrasing, how does this look?